app/main/forms/direct_award_forms.py

```diff
@@ -12,14 +12,43 @@
 
 
 class CreateProjectForm(FlaskForm):
-    name = StripWhitespaceStringField(
-        'Name your search', id="project_name",
+    save_search_selection = RadioField(
         validators=[
-            Length(min=1,
-                   max=100,
-                   message="Names must be between 1 and 100 characters"),
-            Optional(),
-        ])
+            InputRequired("Please choose where to save your search")
+        ]
+    )
+    name = StripWhitespaceStringField(
+        "Name your search. A reference number or short description of what you want to buy makes a good name.",
+    )
+
+    def __init__(self, projects, **kwargs):
+        super().__init__(**kwargs)
+
+        self.save_search_selection.options = [{
+            "label": project["name"] or f"Untitled project {project['id']}",
+            "value": str(project["id"]),
+        } for project in projects]
+        self.save_search_selection.options.append({
+            "label": "Save a new search",
+            "value": "new_search",
+            "reveal": {
+                "question": self.name.label.text,
+                "hint": "100 characters maximum",
+                "name": self.name.name,
```

Comment:
🏆 🏆 🏆 for moving the hint text etc here

Reply:
Soon it will be on the `self.name` field object itself 😄